package.json

```diff
@@ -52,7 +52,6 @@
     "jest": "^29.5.0",
     "jest-junit": "^16.0.0",
     "jest-serializer-html": "^7.1.0",
-    "prop-types": "^15.6.2",
```

Comment:
The `devDependency` should stay since it won't get automatically installed as a `peerDependency`.

Reply:
yeah good catch 👍